src/Compilers/Core/CodeAnalysisTest/RealParserTests.cs

```diff
@@ -17,7 +17,7 @@ public class RealParserTests
         /// in some implementation.
         /// </summary>
         [Fact]
-        public static void TestTroublesomeDoubles()
+        public void TestTroublesomeDoubles()
```

Comment:
>public void TestTroublesomeDoubles() [](http://example.com/codeflow?start=8&length=36)

Is this change and other similar changes necessary?

Reply:
Reverted everything under src/Compilers.